### ISSUE TITLE
Certificate authority (CA) rotation 

### ISSUE THREAD

**Issue by alwaysastudent** [hsiaoairplane/kucero#14]
Does this project help doing CA certificate rotation? 

https://kubernetes.io/docs/tasks/tls/manual-rotation-of-ca-certificates/

**Comment by jenting** [comment]
> Does this project help doing CA certificate rotation? 
> 
> 
> 
> https://kubernetes.io/docs/tasks/tls/manual-rotation-of-ca-certificates/

@alwaysastudent no, because kubeadm bootstrapped cluster CA certificate by default is 10 years, so we didn't cover this yet.
